clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -41,7 +41,46 @@
     private volatile long lastRecordTime;
     private final long inactiveSensorExpirationTimeMs;
 
-    Sensor(Metrics registry, String name, Sensor[] parents, MetricConfig config, Time time, long inactiveSensorExpirationTimeSeconds) {
+    public enum RecordLevel {
+        INFO("INFO"), DEBUG("DEBUG");
```

Comment:
Will do this, ideally before feature freeze, but definitely before code freeze. Stay tuned.